psi4/driver/procrouting/proc.py

```diff
@@ -1904,6 +1919,11 @@ def run_occ(name, **kwargs):
 
     occ_wfn = core.occ(ref_wfn)
 
+    # Shove variables into global space
+    if name in ['mp2', 'omp2', 'scs-omp2', 'scs(n)-omp2', 'scs-omp2-vdw', 'sos-omp2']:
```

Comment:
Can this be simplified here and at other similar lines? Seems a bit brittle and easy to miss in the future.

Reply:
It's all the names in the mp2 if/elif block above. No way to accumulate them, really unless `'mp2' in name and not name.endswith('mp2.5')`. only way I see would be to break `run_occ` up into mp2/mp2.5/mp3/ccd `run_` fns. while I wholly agree that it's brittle (along with all of occ), it should at least heal toward removing that `if name in [` line as further wfn psivar passes hit mp3, cc, etc.

other similar lines?

Reply:
Could we have a boolean that is flip above then? Might be less dangerous if we localize the variable change.